Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -576,17 +572,9 @@ class BrowserViewController: UIViewController {
         self.present(alert, animated: true, completion: nil)
     }
 
-    fileprivate func shouldRestoreTabs() -> Bool {
+    fileprivate func canRestoreTabs() -> Bool {
         guard let tabsToRestore = TabManager.tabsToRestore() else { return false }
-        let onlyNoHistoryTabs = !tabsToRestore.every {
-            if $0.sessionData?.urls.count ?? 0 > 1 {
-                if let url = $0.sessionData?.urls.first {
-                    return !url.isAboutHomeURL
-                }
-            }
-            return false
-        }
-        return !onlyNoHistoryTabs && !DebugSettingsBundleOptions.skipSessionRestore
+        return tabsToRestore.count > 0
```

Comment:
I think this is fine. I don't remember why we wanted to not restore blank tabs but I like the idea of getting back exactly what you had open.